app/main/views/frameworks.py

```diff
@@ -1,13 +1,36 @@
 import datetime
 
 from flask import jsonify, abort, request
+from sqlalchemy import func, orm, case, text
 from sqlalchemy.exc import IntegrityError, DataError
-from sqlalchemy import func, orm, case
 from dmapiclient.audit import AuditTypes
 from dmutils.config import convert_to_boolean
 
 from .. import main
-from ...models import db, Framework, DraftService, User, Supplier, SupplierFramework, AuditEvent, Lot
+from ...constants import (
```

Comment:
is this still `constants`, or `supplier_constants` now?

Reply:
Good spot. That's why tests are failing. :man_facepalming: